.github/actions/publish-nuget/New-NuGetPackage.ps1

```diff
@@ -85,6 +88,9 @@ foreach ($project in $projects)
             'have a <PackageLicenseFile> property. You can avoid this check by including the ' +
             '<IsPackable>false</IsPackable> property.')
 
+        Write-Output "isPackableProperty: '$isPackableProperty'"
+        Write-Output (Get-Content $project)
```

Comment:
Is this also needed for the magic fix or is this rather leftover debug code?

Reply:
I left it in as context that can be helpful every time the warning above will be displayed. It's intentionally not part of the warning message but still available. I think we can feel free to log more during warnings and exceptions. Regular operation dumps a lot of text into the action anyway so I see no downside.

Reply:
OK, sounds good.